### ISSUE TITLE
AttributeError: property 'protocol' of 'RemoteBus' object has no setter

### ISSUE THREAD

**Issue by ptoews** [hardbyte/python-can#1809]
### Describe the bug
```
Traceback (most recent call last):
  File "C:\Users\Philipp.Toews\AppData\Roaming\JetBrains\PyCharmCE2024.1\scratches\scratch_56.py", line 3, in <module>
    b1 = can.interface.Bus(bustype="remote", channel="ws://localhost:54701")
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Philipp.Toews\Development\Datalogger\DLv2\venv_312\Lib\site-packages\can\util.py", line 379, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Philipp.Toews\Development\Datalogger\DLv2\venv_312\Lib\site-packages\can\interface.py", line 135, in Bus
    bus = cls(channel, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Philipp.Toews\Development\Datalogger\DLv2\venv_312\Lib\site-packages\can_remote\client.py", line 35, in __init__
    self.protocol = RemoteClientProtocol(config, websocket)
    ^^^^^^^^^^^^^
AttributeError: property 'protocol' of 'RemoteBus' object has no setter
```

### To Reproduce
```
import can

b1 = can.interface.Bus(bustype="remote", channel="ws://localhost:54701")
```

### Additional context

OS and version: Windows 10
Python version: 3.12
python-can version: 4.4.2
python-can interface/s (if applicable): remote

Update: 4.2.2 is the last version where it worked

**Comment by zariiii9003** [comment]
Update python-can-remote. This should be fixed in 0.2.2.

**Comment by ptoews** [comment]
Thanks!
